semantic-python/semantic-python.cabal

```diff
@@ -25,6 +25,7 @@ common haskell
                      , fused-syntax
                      , parsers ^>= 0.12.10
                      , semantic-analysis ^>= 0
+                     , semantic-ast
```

Comment:
Note that this does not, as far as I’m aware, present an obstacle due to per-component builds; `semantic-ast`’s executable component depends on `semantic-python`, but the dependency graph is still acyclic since its library component does not.

Reply:
In order to work around this, I have moved `semantic-ast`’s executable into the new `semantic-parse` package, which can safely depend on the lang-specific packages.